docs/user/getting_started.md

```diff
@@ -0,0 +1,216 @@
+# Basic Usage
+
+To start running Mizar, first you will need a Kubernetes cluster. Running Mizar will  install experimental features on your cluster. **It is recommended that you use a test cluster to run Mizar.**
+
+## Preqrequesites
+
+### Kind (Kubernetes in Docker)
+
+The recommended way to try out Mizar is with Kind.
+Kind can be used to run a multi-node Kubernetes cluster with Docker containers locally on your machine.
+You can find [instructions for installing Kind on it's official site here.](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+### Linux Kernel Update
+
+For TCP to function properly, you will need to update your Kernel version to at least 5.6-rc2 on every node. A script is provided in the Mizar repo to download and update your machine's kernel if you do not wish to build the kernel source code yourself.
+
+### Mizar Compilation Prerequisites
+Make is used to build the entire data-plane code. An Ubuntu-based host is recommended for development and functionality testing. To compile, run Mizar on an existing host, please follow the following steps:
+
+* Clone Mizar
+```
+$ git clone --recurse-submodules -j8 https://github.com/futurewei-cloud/mizar.git
+```
+* Enter the mizar directory and run the bootstrap.sh script.
+    * This script will the neccessary components to compile Mizar, and run it's unit tests. These include
+        * Python
+        * Docker
+        * Build Essentials
+        * Cmocka
+        * Clang-7
+        * Llvm-7
+    * There is also an optional kernel update included in the script if you wish to update to a compatible version
+
+```
+$ ./bootstrap.sh
+```
+
+
+## Setup
+
+### Compiling Mizar
+
+After completing the prerequisites above, you can now compile mizar. Enter the mizar directory and run the command below.
+
+```
+$ make
+```
+
+* You can now optionally run the unit tests and coverage reports by running the commands below.
+
+```
+$ run_unittests
+$ make lcov
+```
+
+### New Kind Cluster
+
+If you are testing out Mizar with Kind, a script has been included to setup a local Kubernetes cluster, and install all of the components needed to run Mizar.
+Simply run the script below in the Mizar directory.
+
+```
+$ ./kind_setup.sh
+```
+
+This script does the following:
+
+* Create a multi-node local Kubernetes cluster with Kind.
```

Comment:
[ADD] this kubernetes cluster is used as a cluster of VM's or droplets, and will later be used to provision test environment